hyperopt/fmin.py

```diff
@@ -281,6 +322,11 @@ def fmin(fn, space, algo, max_evals, trials=None, rstate=None,
         for example if it is expected that `len(trials)` may be zero after
         fmin, and therefore `trials.argmin` would be undefined.
 
+    points_to_evaluate : list, default None
+        If None trials are evaluated normally. If dict with points to
+        evaluate is added the points are evaluated before optimisation starts.
```

Comment:
Can you maybe add an example and/or a more detailed description here what "points" look like? this is what users will ultimately read to use this. Like in your test below.

Reply:
ok